src/borg/archiver.py

```diff
@@ -3525,8 +3530,12 @@ def define_borg_mount(parser):
                                help='do not actually change any files')
         subparser.add_argument('--numeric-owner', dest='numeric_owner', action='store_true',
                                help='only obey numeric user and group identifiers')
+        # --nobsdflags is misleading and deprecated.
+        # use --noflags as a replacement.
```

Comment:
as this is now in help text, maybe we do not need the comment at all.

Reply:
Removed, too.